install_config/persistent_storage/topics/glusterfs_prereqs_hardware.adoc

```diff
@@ -0,0 +1,30 @@
+Any nodes used in a {gluster-native} or {gluster-external} cluster are
```

Comment:
For the purposes of the documentation we call them storage nodes. Regardless of whether they are designated as app, infra, or otherwise, if they provide storage they are storage nodes. This is not a codified label, this is just a convention.